config-connector/solutions/iam/kpt/storage-bucket-iam/README.md

```diff
@@ -0,0 +1,37 @@
+storage-bucket-iam
+==================================================
+
+# NAME
+
+  storage-bucket-iam
+
+# SYNOPSIS
+
+  Config Connector compatible yaml to enable permissions for a storage bucket.
+
+# REQUIREMENTS
+
+  A working Config Connector cluster using a service account with the following roles:
+
+  Note: Using uniform bucket-level access control is recommended for this package.
+
+# USAGE
+
+  Set the name of the bucket you want to configure permissions for:
+  ```
+  kpt cfg set . bucket-name your-bucket
+  ```
+  Set the [storage
+role](https://cloud.google.com/iam/docs/understanding-roles#storage-roles)
+(defaults to `roles/storage.objectViewer`) that you want to apply and the IAM
+member the role will apply to.
+  ```
+  kpt cfg set . iam-name user:name@example.com
+  kpt cfg set . role roles/storage.admin
+  ```
+  Once the configuration is satisfactory, apply:
+  ```
+  kubectl apply -f .
+  ```
+
+# SEE ALSO
```

Comment:
Any reason you add this section?

Reply:
It's one of the defaults kpt creates when initializing a package.